libraries/AP_Motors/AP_MotorsMulticopter.h

```diff
@@ -126,11 +127,17 @@ class AP_MotorsMulticopter : public AP_Motors {
     // return gain scheduling gain based on voltage and air density
     float               get_compensation_gain() const;
 
-    // convert thrust (0~1) range back to pwm range
-    int16_t             calc_thrust_to_pwm(float thrust_in) const;
+    // convert actuator output (0~1) range to pwm range
+    int16_t             output_to_pwm(float _actuator_output);
+    
+    // converts desired thrust to linearized actuator output in a range of 0~1
+    float               thrust_to_actuator(float _actuator_output);
+    
+    // adds slew rate limiting to actuator output if MOT_SLEW_TIME > 0 and not shutdown
+    void                set_actuator_with_slew(float& actuator_output, float input);
 
-    // calculate spin up to pwm range
-    int16_t             calc_spin_up_to_pwm() const;
+    // gradually increase actuator output maximum limit
+    float               actuator_spin_up() const;
```

Comment:
I wonder if we could come up with a more easily understood name and/or comment for the "actuator_spin_up()" function.  maybe it's "actuator_spin_up_to_ground_idle()".  That's a bit long but "spin_up" alone doesn't really clarify that it's only spinning up to ground-idle.